OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -182,13 +173,25 @@ Action InitDisplayPanel(Widget panel)
 			var cursorDoubleIsChecked = cursorDoubleCheckbox.IsChecked;
 			cursorDoubleCheckbox.IsChecked = () => !cursorDoubleCheckbox.IsDisabled() && cursorDoubleIsChecked();
 
-			panel.Get("WINDOW_RESOLUTION").IsVisible = () => ds.Mode == WindowMode.Windowed;
+			panel.Get("WINDOW_RESOLUTION").IsVisible = () => ds.Mode != WindowMode.NativeFullscreen;
```

Comment:
Erm, what's going on here?  The resolution control only makes sense for the windowed mode.